framework/src/main/java/org/checkerframework/common/basetype/BaseTypeChecker.java

```diff
@@ -619,10 +622,29 @@ private CheckerMessage(
                 String message,
                 @FindDistinct Tree source,
                 @FindDistinct BaseTypeChecker checker) {
+            this(kind, message, source, checker, null);
+        }
+
+        /**
+         * Constructor method for checker message if stack trace needs to be stored.
```

Comment:
Don't say `constructor message`. The same `Create a new CheckerMessage.` as for the other constructor is fine.

Reply:
Actually, there are no calls of the constructor without a trace, so why have two constructors?

Reply:
I have updated the summary and removed the first redundant constructor from BaseTypeChecker.